parallel-consumer-core/src/test/java/io/confluent/csid/utils/TrimListRepresentationTest.java

```diff
@@ -0,0 +1,44 @@
+package io.confluent.csid.utils;
+
+/*-
+ * Copyright (C) 2020-2022 Confluent, Inc.
+ */
+
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+/**
+ * @author Antony Stubbs
+ * @see TrimListRepresentation
+ */
+@Tag("transactions")
```

Comment:
why is it tagged as Transactions? i can see its moved from Transaction related test class but they seem to be completely isolated utility tests?

Reply:
PR was created after, the refactor is apart of the transactions work. but you're right - I'll remove it.